build.gradle.kts

```diff
@@ -45,31 +72,40 @@ java {
   targetCompatibility = JavaVersion.VERSION_14
 }
 
+java {
+  sourceSets["main"].java {
+    srcDir("${buildDir}/generated/sources/srcgen")
+    srcDir("${buildDir}/generated/logisim/")
+  }
+}
+
 task<Jar>("sourcesJar") {
   group = "build"
-  description = "Creates a source jar archive."
+  description = "Creates a source JAR archive file."
```

Comment:
`source JAR archive` is a kind of file, so the "file" at the end of the line is redundant.